scripts/publish.js

```diff
@@ -0,0 +1,37 @@
+const { argv } = require('process');
+const utils = require('util');
+const exec = utils.promisify(require('child_process').exec);
+
+async function publish(publishType) {
+  if (!publishType || !publishType.match(/major|minor|patch/)) {
+    console.error(
+      'Invalid publish type: Available types are major|minor|patch'
+    );
+    process.exit(1);
+  }
+
+  try {
+    /* 1.  Make sure the bump type is correct */
+    await exec(`npm version ${publishType} --git-tag-version=false`);
+
+    /* 2.  Build and test with the correct version. Building the library will also update the src/info.ts file */
+    await exec('yarn build');
+    await exec('yarn test');
+
+    /* 3.  Git actions */
+    await exec('git add -A');
+    await exec(
+      `git commit -am "Bump release ${require('../package.json').version}"`
```

Comment:
🙃 We can follow the conventional commits convention and name this commit as `build: x.y.z`

Reply:
Done